### ISSUE TITLE
Min Max Guage settings

### ISSUE THREAD

**Issue by mastrond** [colinl/node-red-dashboard-2-ui-gauge-classic#9]
Can you give an example of how you update the min and max value dynamically.
I'm able to change the labels and units but not the range for min and max.

**Comment by colinl** [comment]
I am on the road at the moment, it is exactly the same way but with properties min and max. Did you restart node red and refresh the browser page?

**Comment by colinl** [comment]
Example
~`msg.payload.ui_update = {min: 0, max: 10}`~
[Edit] that should be `msg.ui_update = {min: 0, max: 10}`

Did you get it going?

**Comment by mastrond** [comment]
Hi Sorry had a family function I had to attend to.
I love this classic gauge. I just hope I can get this feature to work.
Setting the segment bar colors works perfect so far and changing titles. 
Min Max still not working. 
I still cannot get it going.
I did your suggestions restarted node red and refreshed the browser.
This works
![injectnode](https://github.com/user-attachments/assets/eeda8279-47d9-4676-9faa-ea1ed9f5c366)

These Do Not work
![injectnode1](https://github.com/user-attachments/assets/8fa1652e-ea2c-4427-97d9-82ee2369f67b)
![injectnode2](https://github.com/user-attachments/assets/9c8f0a8a-11e7-4f09-92ff-5a7c922e73db)

**Comment by colinl** [comment]
In the change nodes it should not be an array, so just `{"min",10}` not `[{"min",10}]`. The sectors one is an array because it is an array of sectors.
The function code not working is my fault, I mistyped the example, it should be
`msg.ui_update = {min: 0}`
or
`msg.ui_update = {min: 0, max: 10}`

I have corrected my post above

**Comment by mastrond** [comment]
Sorry I'm just so confused.   I have downloaded your example codes from github  and they work great. None of those examples have the min and max value change.
Is it possible to send me a flow that has the ability to change min and max.   For some reason I'm just not getting it to work.
thanks in advance.

**Comment by colinl** [comment]
Feed this into a gauge, Deploy, and click it to change the range
```
[{"id":"798e7a596307a7c9","type":"inject","z":"997da33a0beedade","name":"min -1, max 9","props":[{"p":"ui_update","v":"{\"min\": -1, \"max\": 9}","vt":"json"},{"p":"topic","vt":"str"}],"repeat":"","crontab":"","once":false,"onceDelay":0.1,"topic":"","x":710,"y":6020,"wires":[["6737de28bc7b6e8e"]]}]
```
If it still doesn't work then export the inject node and your gauge and paste them here.
Also open the developer tools console in the browser window showing the gauge, refresh the page, and click the inject. See if there are any errors shown in the console when you click the inject.
Also check in manage palette that it is showing version 1.2.0 for the gauge-classic node.

**Comment by mastrond** [comment]
Okay still no go here is the flow I'm using you inject node that you sent.
[flows (54).json](https://github.com/user-attachments/files/16325009/flows.54.json)

There is no errors that I see in the developer tools console

I added the image to the version I'm on.   When I goto  manage palette this image shows my version.  Which is not version 1.2.0
I tried to update but it only gives me this version.
![image](https://github.com/user-attachments/assets/29f44b10-eafa-4e46-8404-eb10d316fdc6)

Hmm How do I update to version 1.2.0 .   This could be the whole issue.
